auth/api/iam/openid4vp.go

```diff
@@ -137,8 +115,7 @@ func (r Wrapper) handleAuthorizeRequestFromHolder(ctx context.Context, verifier
 	}
 
 	session := OAuthSession{
-		ClientID:          walletID,
-		IssuerURL:         oauthIssuer.String(),
+		ClientID:          params.get(oauth.ClientIDParam),
```

Comment:
should check that client_id is not empty, it is a required field

Reply:
1 level higher checks that `client_id` starts with `did:web`. This means it can't be empty right now, but that check is likely to disappear soon

Reply:
is checked in `jar.Parse`